docs/rpc-ceph-install.md

```diff
@@ -0,0 +1,314 @@
+
+Example of an ansible inventory file
+```bash
+[all]
+ceph-stor01
+ceph-stor02
+ceph-stor03
+ceph-stor04
+ceph-stor05
+
+[all:vars]
+ansible_python_interpreter=/usr/bin/python3
+
+[mgrs]
+ceph-mon01
+ceph-mon02
+ceph-mon03
+
+[mons]
+ceph-mon01
+ceph-mon02
+ceph-mon03
+ 
+[osds]
+ceph-stor01
+ceph-stor02
+ceph-stor03
+ceph-stor04
+ceph-stor05
+ 
+[rgws]
+ceph-mon01
+ceph-mon02
+ceph-mon03
+ceph-mon04
+ceph-mon05
+```
+
+### Verify ansible connectivity to all hosts:
+Run an ansible ping
+```bash
+ceph-ansible -i {inventory file} all -m ping
+``` 
+Each server should respond "pong". 
+
+Depending how ssh keys were distributed, you may need to confirm adding host keys to known hosts.
+If necessary, make any corrections (perhaps errors in /etc/hosts and/or the inventory file),
+and re-run the ansible ping command until all servers are reachable without any further interaction. If your OS has default firewall rules, they must be modified to allow ceph traffic.  
+ 
+Other firewall implementations will require other configuration mechanisms, and
+of course more specific firewall configuration may be required. Some general guidelines:
+* mons listen on port 6789
+* osds listen on port numbers starting at 6800 (6801, 6802, 6803, etc.  One port per osd)
+* rgws listens on port 8080 by default, but may be configured for any desired port
+* mrgs services:
+  * dashboard listens on port 7789
+  * restful listens on port 8789
+
+
+### Edit parameter files
+You can make changes to the configration parameter by editing the **playbooks/group_vars/[service]/00-default.yml** files. These files will not be overwritten when git pull request is run.  
+
+#### Edit the playbook/group_vars/all/00-defaults.yml file:
+The file is well documented with comments and default values.  Several RPC specific settings have already been put in place and any customer specific changes should be placed here.
+The most common values which need to be modified are:
+```bash 
+monitor_interface: eth3           # Must specify the physical interface that monitors use to communicate with the "Storage Front-end" network
+public_network: 192.168.0.0/24    # The IP network used on the "Storage Front-end" network
+cluster_network: 192.168.200.0/24 # The IP network used on the "Storage Back-end" network
+ ```
+
+
+#### Edit the playbooks/group_vars/mons/00-defaults.yml file:
+The file is well documented with comments and default values. RPC and customer specific configurations should be placed here.
+```bash 
+Assuming this cluster will be used by Openstack, set "openstack_config" to true so that common openstack pools and users are
+auto-created for us:
+ 
+openstack_config: true
+ 
+There are other openstack related options here, which you may wish to review, but in most cases, no further changes
+are needed.
+ 
+If this ceph cluster will not be used by openstack, then it is likely that no changes to this file are needed.
+```
+
+#### Edit the playbook/group_vars/mgrs/00-defaults.yml 
```

Comment:
We should never edit the defaults file.
Instead creating a playbook/group_vars/mgrs/overrides.yml

Same as below - the 00-defaults.yml file should literally never be changed to avoid overwriting existing customer settings when we update the rpc-ceph repo.

Reply:
Also that is referenced a few times below so just change it each time you say edit 00-defaults.yml for each group_var.

Reply:
I reworked the entire parameter files section to mention the information in each sections (mgrs, mons, rgws, osds) [section].yml.sample file and that any customer specific changes should be placed in a playbooks/group_vars/[section]/overrides.yml file

Reply:
Fixed